fractor/src/Application/Contract/ConfigurableFractorRule.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\Fractor\Application\Contract;
+
+use Symplify\RuleDocGenerator\Contract\ConfigurableRuleInterface;
+
+interface ConfigurableFractorRule extends FractorRule, ConfigurableRuleInterface
+{
+    /**
+     * @param mixed[] $configuration
```

Comment:
The configuration should always be a map of string => mixed, right? Maybe we can at least have that little type info here.

Reply:
It can also be an integer IMHO